TypeCobol/Compiler/Scanner/Scanner.cs

```diff
@@ -413,13 +422,30 @@ public static void ScanTokensLineContinuationGroup(IList<TokensLine> continuatio
 
                             // If an alphanumeric literal that is to be continued on the next line has as its last character a quotation mark in column 72, 
                             // the continuation line must start with two consecutive quotation marks.
+                            // In ColumnsLayout.CobolReferenceFormat we just check that the quotation mark is the last character of the line.
                             if (lastTokenOfConcatenatedLineSoFar.HasClosingDelimiter)
                             {
-                                if ((startOfContinuationIndex + 1) > lastIndex || line[startOfContinuationIndex + 1] != lastTokenOfConcatenatedLineSoFar.ExpectedClosingDelimiter)
+                                if (format == ColumnsLayout.CobolReferenceFormat 
+                                    ? (((startOfContinuationIndex + 1) <= lastIndex && line[startOfContinuationIndex + 1] == lastTokenOfConcatenatedLineSoFar.ExpectedClosingDelimiter) &&
+                                    (lastTokenOfConcatenatedLineSoFar.EndColumn + 8) != 73)
+                                    : ((startOfContinuationIndex + 1) > lastIndex || line[startOfContinuationIndex + 1] != lastTokenOfConcatenatedLineSoFar.ExpectedClosingDelimiter))
                                 {
-                                    continuationLine.AddDiagnostic(MessageCode.InvalidFirstTwoCharsForContinuationLine, startOfContinuationIndex, startOfContinuationIndex + 1, lastTokenOfConcatenatedLineSoFar.ExpectedClosingDelimiter);
+                                    continuationLine.AddDiagnostic(MessageCode.InvalidFirstTwoCharsForContinuationLine, startOfContinuationIndex, startOfContinuationIndex + 1, lastTokenOfConcatenatedLineSoFar.ExpectedClosingDelimiter,
+                                        format == ColumnsLayout.CobolReferenceFormat
+                                        ? "in column 72"
+                                        : "at the last column");
```

Comment:
I'm not sure we should check anything in FreeReferenceFormat because continuation is not needed in this format.
@smedilol what do you think ?

Reply:
I didn't manage to find specifications for free format and as we don't use this format in our company, I suggest to keep it like that.